setup/_tools.py

```diff
@@ -5,6 +5,11 @@
 import json
 import os
 
+tools = None
+data_file = os.path.join(os.path.dirname(__file__), "_tools.json")
+with open(data_file, "r") as f:
+    tools = json.load(f)
```

Comment:
Cool - I've always used `json.loads(f.read())`, good to know there's an option which takes a file handle.